solidity/deploy/22_retarget_relay.ts

```diff
@@ -0,0 +1,115 @@
+    "eab1c47040b0f8c3e6740e040000000000000000009e2f76bcf1b3692c48e933575521fd" +
+    "bf1d5d385184823dcd92b8dea27bfb870b549ba06390f507171cc5fc0f0020002007b658" +
+    "524aad00f174782ab83699b043d5358b27656106000000000000000000ebdaba52af764a" +
+    "31f84302bcb544b4fd82fdb33533e01c98fc685854677a4244909ba06390f507177940da" +
+    "6204200020c10e26bc114630b6d90915d744244c97816b971741f0010000000000000000" +
+    "00c00a12d565e8f146b2b2a76918e76a6c89d7e136eaf2c75e6f0214a3df96b769749ca0" +
+    "6390f507175f1b00250020012041c31f14f583f93ddd8e2308e0784398ddcd1360a58b07" +
+    "00000000000000000075a57e4e79adb7c5dbae3b4c4500ed94f7d34a2f1c8dd07d07054e" +
+    "0a1073e1aebe9ca06390f507175958878200605828f12c983c15b919c03089cf117cdc65" +
+    "d865ae88a596e2020000000000000000005a2d55357c0846bfcf695343ab56c270ed1c4c" +
+    "7c116177afd7dc929478b451bde29fa06390f5071744765e2d00801220ffbc53670b79fd" +
+    "a367c6c925d1c2621fa22a42203fc501000000000000000000f0537fc7737c6940af5d17" +
+    "eb388c40189d04d6eb0125ead3cbd092bef9fa590683a0a06390f50717666a1066040060" +
+    "20de7f707e7f0910a44ab8d2e31a9644c70f85fbec4ad20000000000000000000044371c" +
+    "0af4f1a8bbcd61ae8ef65ef4a31c934202d92b05fb0f752534f5352f023fa2a06390f507" +
+    "177f2943b900c00c28b42b6d8f97ce6a82016fa4afd817ac4755cceacb3a100000000000" +
+    "0000000000b9727dd5bcf2870b225edd881106822b1aa353c04e977dc98d119a9dc74dc9" +
+    "e113a7a06390f507172c78994a00007c25e5d58a72ca5e3514bd9bdf48ef57a3e6fafcea" +
+    "4bc44306000000000000000000ed4b8a5f8e44c9e2ad7d9690c80de3c064f1035255ae10" +
+    "c5ae91cdec6ff44c518ca7a06390f50717212a2f6600a08b28d7a6ad25687cde3703a9d9" +
+    "a0f0bee090bd6011fb9a6f040000000000000000007b3261cfb7bb9c78c89792556f1e21" +
+    "b2dd7fc41299d5372d892f08e3fe2a20f7d9ada06390f5071737ef232b0400c020b5418d" +
+    "62d0a19c2de1c96fefe82092339c44183bafa402000000000000000000ac6ae399a73436" +
+    "3043026f008651a3cbc61f6f5cd387428d70638f89d1b86238e9aea06390f507174772b7" +
+    "23000080200941f2c5fd01cc7dbee913617861ef759c56e9593939010000000000000000" +
+    "000adb93b71075857413826d55078d633d7106f433e8e38bb36b428b7608cf539e3bb6a0" +
+    "6390f507170444c90000004020fee3c940a2f19e5f04a57c668388ebfab54dbb8b4d3602" +
+    "000000000000000000319cdaf0010646d09750bf4f7ffeee6a593e296145ca1529b90924" +
+    "9a5f6bbf9ac5b8a06390f507176e386e800000a020662cdabd29bad13621fca6ee601ceb" +
+    "6cea36a0c25fce00000000000000000000d46e68944e26294232b97110be78c2a9afe032" +
+    "34e0baba2e23595e095d1948b9f6b8a06390f5071763ba125100e0ff3f5cb5f2e1e73b9b" +
+    "a9382b47ce261e12c37b211e5a81b600000000000000000000b6bcda49736fb4a79f2b60" +
+    "320904198cdd0f703cf25a4dcece27aafd6429a76916bfa06390f50717060b6d11"
+
+  await execute(
+    "LightRelay",
+    { from: deployer, log: true, waitConfirmations: 1 },
+    "retarget",
+    headers
+  )
+}
+
+export default func
+
+func.tags = ["RetargetLightRelay"]
+func.dependencies = ["LightRelay"]
```

Comment:
Shouldn't it depend on `GenesisLightRelay`?

Reply:
It should.
Fixed in e652e7e61f070fa573232761b15a87adddb5adc8.